### ISSUE TITLE
[Feature request] Expose command being ran

### ISSUE THREAD

**Issue by nunocastromartins** [CyberShadow/aconfmgr#70]
Hi, I'm creating an issue to specifically track a solution for a question made in [this comment](https://github.com/CyberShadow/aconfmgr/issues/23#issuecomment-356410282):
> Is there a way to make sure that this only happens on apply, and never on save?

I have a few use cases where knowing which command is being ran (`save` or `apply`) would make sense.

As an example, it would be useful in order to solve #33 (syncing the package db after changing `pacman.conf`). My current fix for the issue is as follows:
```bash
# Sync the package databases when the pacman config changes
cmp /etc/pacman.conf "$config_dir"/files/etc/pacman.conf >/dev/null 2>&1
pacmanconf_cmp_status=$?
CopyFile /etc/pacman.conf
if [[ $pacmanconf_cmp_status != 0 ]]; then
  sudo pacman -Sy
fi
```

However, the syncing will run for both `save` and `apply` when the file changes. I would thus like the code to be something like the following (so that it would only sync during an `apply` operation):
```bash
# Sync the package databases when the pacman config changes
cmp /etc/pacman.conf "$config_dir"/files/etc/pacman.conf >/dev/null 2>&1
pacmanconf_cmp_status=$?
CopyFile /etc/pacman.conf
if [[ $pacmanconf_cmp_status != 0 && $aconfmgr_command == apply ]]; then
  sudo pacman -Sy
fi
```

A workaround I found was to check whether the variable `config_save_target` was set (it is only set during the `save` command). However, besides it being a very hacky solution, it doesn't work as intended when running the `check` command (since the variable is also not set there).

**TLDR:** Please expose an `aconfmgr_command` variable (or something similar) that would have as value `"apply"`, `"save"`, or `"check"` depending on the command being ran.

Thank you for your work! :)

**Comment by CyberShadow** [comment]
Does 8f5233291ed501f812803f07de0503ea3f79422a work for you?

One more action that may be added in the future is `diff`, which makes no changes to the system or configuration, but describes the differences between them.

**Comment by nunocastromartins** [comment]
Perfect! Thank you for the quick fix. :)

A `diff` action sounds useful too. Again, thank you for your work!
